### PR TITLE
Add direct OkHttpClientFactory usage (#12)

### PR DESCRIPTION
**Pull request by jchipmunk**
This PR adds direct `OkHttpClientFactory` usage. This solves the issue #12 with loading the `HttpClient` implementation when kafka-kubernetes-config-provider is deployed in the directory specified in the `plugin.path` property.

Signed-off-by: jchipmunk <andrey.pustovetov@gmail.com>

### CODE REVIEW

**Review by scholzj — COMMENTED** [review]
Please see the comment on the issue.

File: pom.xml
Comment (scholzj) [reply pom.xml:181]: @sonatype-lift ignore

**Review by scholzj — COMMENTED** [review]
Ok, I let's do this as discussed ont he issue. Could you please fix the DCO signoff? The instructions should be under the _Details_ link next to the DCO status.

**Comment by scholzj** [comment]
@jchipmunk Can you please fix the DCO signoff? We cannot merge it without it.

**Comment by jchipmunk** [comment]
> @jchipmunk Can you please fix the DCO signoff? We cannot merge it without it.

Thanks for the review. I have fixed the DCO signoff, the PR can be merged.

**Review by scholzj — APPROVED** [review]
Thanks.

**Comment by scholzj** [comment]
@strimzi/maintainers Can we get a second review please?

**Review by Frawless — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks for the PR.

**Pull request merged**
